terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/redirect/RedirectTest.java

```diff
@@ -33,6 +33,9 @@
 
     @Value("${app.redirect.allowed.externalUrl}")
     String redirectionAllowedExternalUrl;
+    
+    @Value("${app.redirect.pageTitle.404Error}")
+    String pageTitle404Error;

```

Comment:
@btyoshikawaa 
I think it does not make sense. This change is just removing hard coding and putting it on properties. It looks a refactoring and assertion is not changed completely.
I thinks it's OK for this PR contains tiny refactoring and I will merge, but I'm just afraid you have some mis-understandings.